src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
```

Comment:
I view things like this as one conceptual statement and find it takes more mental effort to have the line break than not for single statements, particularly with case statements as my mind already is prepping for a multi-line statement if there's a line break after the rocket.

I'd much prefer to go by "is this describing a single thing" instead of slavishly adhering to a fixed number. If you're starting to get up in the 120 char range it's a sign that perhaps you're chaining too many complicated things together  but these were fairly simple constructs